src/net.cpp

```diff
@@ -45,6 +45,8 @@
 
 // We add a random period time (0 to 1 seconds) to feeler connections to prevent synchronization.
 #define FEELER_SLEEP_WINDOW 1
+/** Run the feeler connection loop once every 2 minutes or 120 seconds. **/
```

Comment:
Comment should reflect that FEELER_INTERVAL has been changed to 1 minute/60 seconds.